www/waterfall_view/src/module/main.module.coffee

```diff
@@ -104,6 +107,18 @@ class Waterfall extends Controller
             # Bind scroll event listener
             angular.element(containerParent).bind 'scroll', onScroll
 
+            @$window.onkeydown = (e) =>
+                # +
+                if e.keyCode is 107
```

Comment:
I would expect this to update the settings as well, so that you adjust the settings with keyboard